pkg/daemon/ceph/osd/daemon.go

```diff
@@ -281,19 +281,27 @@ func Provision(context *clusterd.Context, agent *OsdAgent, crushLocation string)
 
 func getAvailableDevices(context *clusterd.Context, desiredDevices []DesiredDevice, metadataDevice string, pvcBacked bool) (*DeviceOsdMapping, error) {
 
+	var partCount int
```

Comment:
nit: Why do you need to declare these four vars above? Looks like they're just used in the scope of line 291.

Reply:
Because L356 needs it, https://github.com/rook/rook/pull/4682/files#diff-d1ca60bbccae22948f1847a6b9747cd5R364.

Reply:
ok, maybe just move them inside the for loop